_posts/2021-01-31-choosing-randomization-unit.md

```diff
@@ -0,0 +1,353 @@
+---
+layout: post
+title: Choosing your randomization unit in online A/B tests
+author: ianwhitestone
+summary: A discussion around the implications of different randomization units in online controlled experiments
```

Comment:
"A discussion around the implications of session versus user level randomization in online controlled experiments"

☝️ switch to this if you drop the stuff around metrics